packages/dcos-net/extra/dcos-net-setup.py

```diff
@@ -15,6 +15,8 @@
 The script allows to add configuration for networkd
 """
 
+import datetime
```

Comment:
Should we consider this code critical and cover it with some `integration-tests`? One idea would be to spin up containers with all the linux versions we support, and tests that this can drive a dummy systemd unit.

Reply:
Valid point, but I do not think that this fits the scope of this PR :)

Reply:
This script already has 90%+ code coverage by integration tests, you run it every time when you deploy dc/os. We may consider adding some unit tests for failing path, but, as @vespian said, it's out of scope for this PR.